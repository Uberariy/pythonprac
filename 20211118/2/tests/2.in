class Cd:
		num = Num()

print(Cd().num)
c, d = Cd(), Cd()
c.num = d.num = 20
print(c.num+d.num)
c.num = ""
print(c.num+d.num)
d.num = range(10, 10000, 19)
print(c.num+d.num)

