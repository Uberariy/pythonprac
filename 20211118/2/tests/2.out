0
40
20
526
